lib/devise-security/models/session_limitable.rb

```diff
@@ -12,10 +12,16 @@ module Models
     module SessionLimitable
       extend ActiveSupport::Concern
 
+      # Update the unique_session_id on the model.  This will be checked in
+      # the Warden after_set_user hook in {file:devise-security/hooks/session_limitable}
+      # @param unique_session_id [String]
+      # @return [void]
```

Comment:
`null` over `void` in ruby?

Reply:
@return [void] is yard's way of indicating that the return value is not significant.